.github/workflows/test-everything.yml

```diff
@@ -92,11 +92,15 @@ jobs:
         # See https://github.com/open-mpi/ompi/issues/6518
         OMPI_MCA_btl: "self,tcp"
     steps:
-      - name: Get clang/gcc 8
+      - name: Get clang/gcc 8, libxml2, Linux
```

Comment:
Can we rename to "Linux: get clang/gcc 8, libxml2" (and "MacOS: get libxml2" below)

Reply:
Sure!